src/views/CartoonPage.vue

```diff
@@ -78,12 +78,23 @@ export default {
     video_url: function () {
       let url = this.data.video;
       try {
-        url = require("@/assets/" + url); // match the url and use that image
+        if(url.length>0)
+          url = require("@/assets/" + url); // match the url and use that video
       } catch (e) {
-        console.log(e);
+        url=""; //return empty url in case it is inexistant
       }
       return url;
     },
+    video_mime: function(){
```

Comment:
in this case CartoonPage.vue don't need line 99 to 112 already right?

Reply:
still use it in line 22 for the video poster
`<video controls :poster="cover(data.thumbnail)">`

Reply:
Oh shoot you're right, we'll remain then 👍🏻👍🏻👍🏻